### PR TITLE
Access role filter

### PR DESCRIPTION
**Pull request by akshat-rippling**

### CODE REVIEW

File: cloudlift/utils/yaml_parser.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+   def __iter__(self):
+       for item in self.items:
+           if isinstance(item, list):
+               for nested_item in item:
+                   yield nested_item
+           else:
+               yield item
+
+def config_keys(access_role):
+    data = yaml.load(Path('access.yml'))
```
Comment (siliconsenthil) [inline cloudlift/utils/yaml_parser.py:27]: This could be made configurable?

**Review by siliconsenthil — COMMENTED** [review]
Can you add a test covering this flow with a sample yaml file.

File: cloudlift/utils/yaml_parser.py (hunk 1 above)
Comment (akshat-rippling) [reply cloudlift/utils/yaml_parser.py]: Made this configurable.

**Comment by akshat-rippling** [comment]
> Can you add a test covering this flow with a sample yaml file.

@siliconsenthil Added a test

**Review by siliconsenthil — APPROVED** [review]

**Pull request merged**